docs/Windows.md

```diff
@@ -1,43 +1,172 @@
-#### Installing the nightlies
+The following commands are to be executed from within this Native Tools Command Prompt. Be sure to always start the `x64` version of the Native Tools Command Prompt.
+
+#### Files for the Windows SDK
+
+You will need to copy a few files into the Windows SDK to make it usable for Swift development. As you might need  administrator rights to copy files into the Visual Studio installation directory, for the following four commands (and for those commands only) open the Native Tools Command Prompt with administrator rights. Otherwise, you might get a "Permission denied" error for some of those commands. Be sure that all commands have been successfully executed before continuing.
+
+To open the Native Tools Command Prompt with administrator rights, navigate to the according entry in the start menu, right-click on that entry and choose "Open location" in the context menu. Then right-click the according file (that is actually a link) and choose "Run as administrator".
+
+Then execute the following four commands from within the Native Tools Command Prompt:
 
-1. Extract the toolchain artifact.  We assume that the toolchain will be installed to `C:\Library`.  Extracting it to that location should give you a hierarchy that looks like `C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin\swiftc.exe`.
-2. In order to develop with this toolchain, you will need an installation of the Windows SDK.  The easiest way to do this is to install Visual Studio.  Additionally, you will need to copy a few files into the SDK to make it usable from swift.
 ```cmd
 curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/ucrt.modulemap" -o "%UniversalCRTSdkDir%\Include\%UCRTVersion%\ucrt\module.modulemap"
 curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/visualc.modulemap" -o "%VCToolsInstallDir%\include\module.modulemap"
 curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/visualc.apinotes" -o "%VCToolsInstallDir%\include\visualc.apinotes"
 curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/winsdk.modulemap" -o "%UniversalCRTSdkDir%\Include\%UCRTVersion%\um\module.modulemap"
 ```
-3. You will need to add the ICU libraries for the target.  The nightlies are built against ICU 64.2 from the ICU project.  You can download the binaries for that from http://download.icu-project.org/files/icu4c/64.2/icu4c-64_2-Win64-MSVC2017.zip.
+
+Close this instance of the Native Tools Command Prompt after those commands have been successfully executed.
+
+#### Swift Toolchain & Platform SDK concepts
+
+The installation instructions that follow will result in a directory tree that has a well thought-out structure. To understand this directory structure better you may consult  <https://github.com/compnerd/windows-swift/blob/master/docs/details.md>. But first, you might want to continue with the installation.
```

Comment:
I think that the way to link the document is `[the details document](details.md)`.  s/may/can/, and I believe that the last sentence is superfluous.